mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -372,6 +375,60 @@ impl Drop for Nexus {
     }
 }
 
+extern "C" fn nvmf_create_snapshot_hdlr(req: *mut spdk_nvmf_request) -> i32 {
+    debug!("nvmf_create_snapshot_hdlr {:?}", req);
+
+    let subsys = unsafe { spdk_sys::spdk_nvmf_request_get_subsystem(req) };
+    if subsys.is_null() {
+        debug!("subsystem is null");
+        return -1;
+    }
+
+    /* Only process this request if it has exactly one namespace */
+    if unsafe { spdk_sys::spdk_nvmf_subsystem_get_max_nsid(subsys) } != 1 {
+        debug!("multiple namespaces");
+        return -1;
+    }
+
+    /* Forward to first namespace if it supports NVME admin commands */
+    let mut bdev: *mut spdk_bdev = std::ptr::null_mut();
+    let mut desc: *mut spdk_bdev_desc = std::ptr::null_mut();
+    let mut ch: *mut spdk_io_channel = std::ptr::null_mut();
+    let rc = unsafe {
+        spdk_sys::spdk_nvmf_request_get_bdev(
+            1, req, &mut bdev, &mut desc, &mut ch,
+        )
+    };
+    if rc != 0 {
+        /* No bdev found for this namespace. Continue. */
+        debug!("no bdev found");
+        return -1;
```

Comment:
ENOENT ?

Reply:
This function is derived from nvmf_custom_identify_hdlr upstream and that returns -1 for any error.